app/controllers/book_suggestions_controller.rb

```diff
@@ -0,0 +1,12 @@
+class BookSuggestionsController < ApiController
+  skip_before_action :verify_authenticity_token
+
+  def book_suggestion_params
+    params.permit(:author, :link, :title, :user, :editorial, :price, :publisher, :year, :user_id)
+  end
+
+  def create
+    @booksuggestion = BookSuggestion.create(book_suggestion_params)
```

Comment:
If `create` fails this is gonna respond with an ok with data, but the book suggestion wasn't actually created. You'll have to either handle the case where there are errors, or use `create!` instead (it'll throw an exception if it fails). Which one you use depends on the frontend (do they want some particular error depending on what failed? do they only care about the response code?)
Read about [create](https://apidock.com/rails/v4.2.7/ActiveRecord/Relation/create) and [create!](https://apidock.com/rails/v4.2.7/ActiveRecord/Relation/create%21)

Reply:
Also, `@book_suggestion`